### ISSUE TITLE
error node js express start up with missing UL todo list 

### ISSUE THREAD

**Issue by stevenanthonyrevo** [harttle/liquidjs#45]
I ran into an issue trying to get the express sample to start up on my local machine. 

```
module.js:442
    throw err;
    ^

Error: Cannot find module 'any-promise'
  at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
   at Object.<anonymous> (../../Downloads/liquidjs-master/src/render.js:2:17)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

Solved by adjusting required liquidjs path and package json file.

Terminal: 
Check `npm install`
Run `npm install --save liquidjs`

Packaged JSON File:  
Added `"liquidjs": "^2.0.1"`

Replaced in app.js 
`var Liquid = require(../..)` with `var Liquid = require('liquidjs')`

Solved node js error above however the UL list of todos are missing. 

Any help with this would be great! Thanks

**Comment by harttle** [comment]
Seems top-level contents not being rendered as default block. Give it a block name (just like the `footer` block) solves this problem.

Is this a feature provided by shopify/liquid? I'll look into this problem later and add some tests.

**Comment by stevenanthonyrevo** [comment]
Thanks @harttle!  

I've written two pieces to get the todo working properly. 

## app.js 

```
app.get('/', function (req, res) {
  var todos = ['fork and clone', 'make it better', 'make a pull request'];
  res.render('todolist', {
    todos: todos,
    title: 'Welcome to liquidjs!'
  })
})
```

## todolist.liquid 
```
<ul>
 {% for todos in todos %}
     {% if forloop.first %}
	  <p>This collection has {{ forloop.length }} products:</p>
     {% endif %}
     <li>{{ todos }}</li>
 {% endfor %}
</ul>
```

Let me know if you have any changes!

**Comment by harttle** [comment]
Now it works with top-level contents. Besides the `todolist` in `todolist.liquid` should have been written as `todos`.

Published @2.0.2
